builder/ConfTargets.cmake

```diff
@@ -62,12 +62,18 @@ else()
   set ( MFX_1_25_OPTIONS_ALLOWED OFF )
```

Comment:
This name is misleading: it should be `MFX_1_26_OPTIONS_ALLOWED`. Can you, please, fix in separate commit, same PR.